Modules/Core/Services/UserService.php

```
@@ -65,4 +65,14 @@ public function hasMultipleActiveAdmins(): bool
     {
         return User::active()->admin()->count() > 1;
     }
+
+    /**
+     * Get all users.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getAll()
+    {
+        return User::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Core\Models\User' in method 'getAll'. (undefined)

(StaticAccess)